### PR TITLE
Add Brian the Keyboard

### PR DESCRIPTION
**Pull request by NCKiser**

### CODE REVIEW

File: keyboards/jlw/brian_the_keyboard/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+              _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,            _______,  _______,  _______,  _______,            _______
+    ),
+
+    [2] = LAYOUT(
+         _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+              _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,            _______,  _______,  _______,  _______,            _______
```
Comment (lesshonor) [inline keyboards/jlw/brian_the_keyboard/keymaps/default/keymap.c]: No need to hardcode filler `KC_TRNS` layers. For non-dynamic keymaps it's pointless, and dynamic keymaps will automatically fill in "missing" layers at compile-time up to the maximum set in `dynamic_keymap.layer_count`.

...also don't forget to compile your keymaps before opening a pull request.
```suggestion
                 KC_LCTRL,       MO(1),  KC_ENT, KC_SPC, MO(2),          KC_RALT
```

File: keyboards/jlw/brian_the_keyboard/keymaps/vial/keymap.c [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,            _______,  _______,  _______,  _______,            _______
+    ),
+
+    [2] = LAYOUT(
+         _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+              _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+                   _______,            _______,  _______,  _______,  _______,            _______
```
Comment (lesshonor) [inline keyboards/jlw/brian_the_keyboard/keymaps/vial/keymap.c]: See previous suggestion.
```suggestion
```

**Review by lesshonor — CHANGES_REQUESTED** [review]

File: keyboards/jlw/brian_the_keyboard/keymaps/vial/keymap.c (hunk 2 above)
Comment (NCKiser) [reply keyboards/jlw/brian_the_keyboard/keymaps/vial/keymap.c]: Removed trans layers with recent commit

File: keyboards/jlw/brian_the_keyboard/keymaps/default/keymap.c (hunk 1 above)
Comment (NCKiser) [reply keyboards/jlw/brian_the_keyboard/keymaps/default/keymap.c]: Absolutely correct, thank you for bringing that to my attention. This has been resolved in the most recent commit

**Review by lesshonor — APPROVED** [review]

**Pull request merged**